src/actions/AuthActions.js

```diff
@@ -11,11 +10,28 @@ import {
     GUEST_TOKEN_SET
 } from './types';
 
-import {FACEBOOK_AUTH} from '../../env';
+import {FACEBOOK_AUTH, FIREBASE_USER_PASSWORD} from '../../env';
 import {tokens, navKeys} from '../constants';
 
 import {persistor} from '../store';
 
+const userLoginSuccess = (dispatch, email, navigate) => {
+    dispatch({
+        type: FACEBOOK_LOGIN_SUCCESS,
+        payload: {token: email, appReady: false}
+    });
+
+    navigate(navKeys.SEARCH);
+};
+
+const userLoginFailure = (dispatch, navigate) => {
+    dispatch({
+        type: FACEBOOK_LOGIN_FAILURE
+    });
+
+    navigate(navKeys.LOGIN);
+};
```

Comment:
Added helper functions above for success and failure since we'll be calling them in different places.